demisto_sdk/commands/common/errors.py

```diff
@@ -717,6 +718,10 @@
         "code": "IN157",
         "related_field": "script",
     },
+    "command_reputation_output_capitalization_incorrect": {
+        "code": "IN159",
+        "related_field": "outputs",
+    },
```

Comment:
can you please move the `IN158` entry here and put `IN159` entry under it so the order would make sense?